controllers/cloud.redhat.com/providers/featureflags/localfeatureflags.go

```diff
@@ -91,7 +91,7 @@ func (ff *localFeatureFlagsProvider) EnvProvide() error {
 		LocalFFService,
 	}
 
-	if err := providers.CachedMakeComponent(ff.Cache, objList, ff.Env, "featureflags", makeLocalFeatureFlags, false, ff.Env.IsNodePort()); err != nil {
+	if err := providers.CachedMakeComponent(ff, objList, ff.Env, "featureflags", makeLocalFeatureFlags, false); err != nil {
```

Comment:
I know you mentioned the redundancy here in your description but since you're passing in the whole object as the first parameter, do you still need to pass in `ff.Env` as another param? I do see this in various places.